schema/network.yaml

```diff
@@ -68,6 +68,12 @@ definitions:
         type: string
         readOnly: true
         description: Indicate the interface name.
+      status:
+        type: boolean
+        description: Connection status of the interface, true for connected.
+      wan:
+        type: boolean
+        description: Indicate the interface worked as WAN or not (LAN).
```

Comment:
What is the main purpose of attribute `wan`?
Bundle already has been defined all wan interfaces in json file